packages/modules/providers/payment-stripe/src/types/index.ts

```diff
@@ -21,6 +23,10 @@ export interface StripeOptions {
   paymentDescription?: string
 }
 
+export type HandledErrorType = 
```

Comment:
Since this is just an internal data type, it should live inside the Stripe payment provider (i.e. you can put it at the top of `stripe-base`.